tests/main/uc20-snap-recovery-encrypt/task.yaml

```diff
@@ -10,18 +12,23 @@ restore: |
     if [[ -d ./mnt ]]; then
         umount ./mnt || true
     fi
+    umount /dev/mapper/ubuntu-data || true
+    umount /dev/mapper/test-udata || true
 
     cryptsetup close /dev/mapper/ubuntu-data || true
     cryptsetup close /dev/mapper/test-udata || true
 
     if [ -f loop.txt ]; then
         losetup -d "$(cat loop.txt)"
     fi
+    apt remove -y cryptsetup
 
 prepare: |
     echo "Create a fake block device image"
     truncate --size=6GB fake.img
 
+    apt install -y cryptsetup
```

Comment:
could we include this in pkgdb script for each system?

Reply:
So then we dont need to remove it

Reply:
Yes, probably via followup as this is green currently.